ui/components/ui/update-nickname-popover/update-nickname-popover.js

```diff
@@ -20,8 +20,10 @@ export default function UpdateNicknamePopover({
 }) {
   const t = useContext(I18nContext);
 
-  const [nicknameInput, setNicknameInput] = useState(nickname);
-  const [memoInput, setMemoInput] = useState(memo);
+  const [nicknameInput, setNicknameInput] = useState(
+    nickname === null ? '' : nickname,
```

Comment:
is it possible nickname could be undefined as well? This should happen if no nickname is passed to this component. What if we reverse the change to this line and set a default value to nickname on line 15?

Same comment for `memoInput` below.

```
nickname = '',
```

Reply:
While this greatly improve readability, we would still need to handle the case where `nickname` or `memo` are passed in explicitly as `null`. Setting a default value inside of object destructuring will treat `null` as being defined, thus not defaulting it. For example:
```
function foo({ a, b = 2, c }) { console.log(a, b, c); }
foo({a: 1, b: null, c: 3})
// 1 null 3
foo({a: 1, c: 3})
// 1 2 3
```
I think there is a broader issue here: we are allowing those props to be `null | undefined | string`, where null and undefined are treated logically the same. I can add the `nickname = ''` to handle the `undefined` case, though we still need to handle the `null` case (and so I would need to keep the `nickname === null ? '' : nickname` stuff).

Reply:
Also I think this components' `Address` prop should be `PropTypes.string.isRequired`, but that should be done as a seperate issue so that we can keep the scope minimized for this already widely scoped PR =D

Reply:
Ooh right. Passing in null will not take the default value provided. Glad you caught this. What if we used the nullish operator here? 
```suggestion
    nickname ?? ''
```
ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
I've added the handling of `undefined` (since it is *possible*)

Reply:
I'm also not sure if there would be a case that we pass null, but I like that we're keeping this robust. Sweet :)

Reply:
never hurts to make things stronger!

Reply:
> Ooh right. Passing in null will not take the default value provided. Glad you caught this. What if we used the nullish operator here?
> 
> ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

you learn something every day!!